### PR TITLE
[Issue #57] - Imports manual lists from comicrack database into comixed

### PR DESCRIPTION
**Pull request by jmcrfp**
## Status
**READY**

## Migrations
NO

## Description
Imports manual lists from comicrack database into comixed.

### CODE REVIEW

**Review by mcpierce — DISMISSED** [review]
There are merge conflicts. Can you fix them so I can do a review and merge, please?

File: comixed-importer/src/main/java/org/comixed/importer/ImportFileProcessor.java [hunk 1]
```diff
@@ -74,7 +77,7 @@ public void setImportUser(String user) {
    *
    * @throws ProcessorException if a processing error occurs
    */
-  public void process(String source) throws ProcessorException {
+  public void process(String source) throws ProcessorException, ComicException {
```
Comment (mcpierce) [inline comixed-importer/src/main/java/org/comixed/importer/ImportFileProcessor.java]: I don't think we should add the ComicException to the method signature. Instead, can we have the implementation catch the ComicException and rethrow it as a ProcessorException? That way we know each step of the way in the stacktrace where something bad happened.

**Review by mcpierce — CHANGES_REQUESTED** [review]
Just one change requested, all else looks good.

File: comixed-importer/src/main/java/org/comixed/importer/ImportFileProcessor.java (hunk 1 above)
Comment (jmcrfp) [reply comixed-importer/src/main/java/org/comixed/importer/ImportFileProcessor.java]: You're right. Changed it

**Review by mcpierce — APPROVED** [review]

**Pull request merged**